content/wireless.md

```diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
+
+Router and Access Point Recommendations
+
+Many connection issues are caused by router settings.
+
+Make sure your access point is configured for stability and compatibility.
 
-## Basic Troubleshooting
+Use 2.4 GHz channels 1, 6, or 11 to minimize overlap.
 
-If you’re having problems, try these steps first:
+Set the channel width to 20 MHz for crowded networks.
 
-- Try unplugging the wireless router or modem to reboot it.
-- Try airplane mode by pressing <kbd>Fn</kbd>+<kbd>F11</kbd>, waiting 10s, then disabling.
-- Try rebooting the computer.
+For 5 GHz, use an explicit channel instead of “Auto” when troubleshooting.
 
-Some router settings can cause problems. Try adjusting your access point to these settings:
+Use mixed mode (b/g/n/ax) if devices vary by generation.
 
-- WPA2-AES is preferred over WPA/WPA2 mixed mode or TKIP.
-- A channel width of 20 MHz in the 2.4 GHz band is more stable than automatic 20/40 MHz or fixed 40 MHz.
-- Set 2.4Ghz and 5Ghz SSID names differently.
-- Pick a fixed channel.  Use either 1, 6, or 11 in the 2.4 Ghz band, rather than automatic selection.
-- Check if the router is set to N speeds only.  Auto B/G/N is preferred.
-- Lower the max/burst speeds, turn off channel bonding, and reduce channel width. Setting the speed to 600 Mb/s or 450 Mb/s will use spread frequencies to achieve those speeds and may decrease stability. Try setting it to 289/300 Mb/s (N speed) or or 54 Mb/s (G speed).
-- After making these changes, reboot the router.
+If a single device struggles, temporarily set your router to a common mode and test.
 
-If the issues started after you applied updates, try running this command to make sure a bad WiFi driver has not been installed, then reboot your computer:
+Avoid complex access point features such as:
+ -Band steering
+ -Aggressive airtime fairness
+ -Deep MAC filtering
+ -Ensure your device’s MAC address isn’t being filtered.
+  Check with:
+  
+```bash
+ip link show | grep ether
+```
+
+Then confirm that address is allowed in your router’s admin panel.
+
+You can also confirm what channel and frequency your connection is using:
 
 ```bash
-sudo apt remove backport-iwlwifi-dkms
+iw dev
+iwlist wlan0 scan | grep -E 'SSID|Channel|Frequency'
+```
+
+## Device-Level Checks and Commands
+
+These commands help verify whether your wireless card and drivers are functioning correctly.
+
+ip a — List all network interfaces and IP addresses confirms your Wi-Fi interface (usually wlp2s0 or wlan0) is recognized.
+
+iw dev — Show wireless devices and their states.
+
+sudo rfkill list — Check for hardware or software Wi-Fi blocks.
+
+nmcli device status — Check NetworkManager device states.
+
+sudo systemctl restart NetworkManager — Restart the network stack (safe to run anytime).
+
+journalctl -b | grep -i network — View network-related boot logs.
+
+dmesg | grep -i -E 'wifi|wlan|firmware|ieee80211|rtl|brcm|ath' — View driver and firmware kernel logs.
```

Comment:
All of the commands in this section need formatted correctly. One line tells what the command does, and the next line is the formatted command. 
For example:
List all network interfaces and IP addresses confirms your Wi-Fi interface (usually wlp2s0 or wlan0) is recognized.
```bash
ip a
```